src/helix-ui/styles/core/hx-alert.less

```diff
@@ -0,0 +1,29 @@
+hx-alert {
+  background-color: @blue-100;
+  color: @gray-1000;
+  display: none;
```

Comment:
If an `<hx-alert>` is present in the DOM, it should be visible. Closing an alert should remove it from the DOM.